.travis.yml

```diff
@@ -3,16 +3,26 @@ python: 2.7
 
 sudo: false
 
-env:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=pypy
+matrix:
+  include:
+  - python: 2.6
+    env: TOX_ENV=py26
+  - python: 2.7
+    env: TOX_ENV=py27
+  - python: 3.3
+    env: TOX_ENV=py33
+  - python: 3.4
+    env: TOX_ENV=py34
+  - python: 3.5
+    env: TOX_ENV=py35
+  - python: 3.6
+    env: TOX_ENV=py36
+  - python: pypy
+    env: TOX_ENV=pypy
```

Comment:
I don't understand, we shouldn't need to specify the python versions here because `tox` should take care of that. If it doesn't then what is the point of using `tox` at all?

Reply:
i think the reason is that travis does not come with python3.6 pre-istalled. Tox creates a virtualenv from this executable.

Reply:
exactly, but if you're specifying `python: 3.6` then you're telling TravisCI to run in a container that uses Python 3.6 by default.

we're using tox so that we can run Python 2.7 locally (or whatever version you use) and tox will take care of creating a virtualenv with the correct Python version for the test that it currently is running.

Reply:
i thought virtualenv needed an executable of the correction python version?

Reply:
nope, tox should take care of putting the correct python version into the virtualenv